src/Compilers/Core/AnalyzerDriver/AnalyzerManager.cs

```diff
@@ -265,12 +265,6 @@ internal bool IsSupportedDiagnostic(DiagnosticAnalyzer analyzer, Diagnostic diag
 
             foreach (var diag in supportedDiagnostics)
             {
-                if (HasNotConfigurableTag(diag.CustomTags))
```

Comment:
I think we meant NotConfigurable tag to only apply for the diagnostic's severity not whether or not it can be enabled. The doc comment here also mentions the same: http://source.roslyn.io/Microsoft.CodeAnalysis/R/207e57dd0b96bd4b.html

Otherwise marking something not configurable + disabled by default is just dead code.